contracts/database/orm/orm.go

```diff
@@ -133,6 +133,8 @@ type Query interface {
 	WhereIn(column string, values []any) Query
 	// WhereNotIn adds a "where column not in" clause to the query.
 	WhereNotIn(column string, values []any) Query
+	// WhereBetween adds a "where column between x and y" clause to the query.
+	WhereBetween(column string, x, y int) Query
```

Comment:
We also need to support `between 'a' and 'c'` and `between '2024-02-01' and '2024-02-02'`.

```suggestion
	WhereBetween(column string, x, y any) Query
```